### ISSUE TITLE
Invalid weights initialization in DepthwiseConvolutionLayer

### ISSUE THREAD

**Issue by Chunosov** [ARM-software/ComputeLibrary#355]
Weight tensor shape is initialized as 
```
TensorShape shape = in->info()->tensor_shape();
shape.set(Window::DimX, _conv_width);
shape.set(Window::DimY, _conv_height);
TensorInfo info = TensorInfo(TensorShape(shape), in->info()->num_channels(), in->info()->data_type(), in->info()->fixed_point_position());
info.set_quantization_info(_quant_info);
_weights.set_info(std::move(info));
```
so the shape of input tensor is taken as a 'template', but last (3) dimension in input tensor is `batch_size` while the 3-th dimension of depthwise convolution weights should be `channel_multiplier`.

**Comment by Chunosov** [comment]
Seems that this `channel_multiplier` should be layer parameter, like `ofm` parameter of `ConvolutionLayer`, it acts the similar way: `out_channels = in_channels * channel_multiplier` (like here https://www.tensorflow.org/api_docs/python/tf/nn/depthwise_conv2d)

**Comment by GeorgeARM** [comment]
@Chunosov yes this is a bug.
Fixed internally and will be part for the next release. We won;t add the channel multiplier parameter yet though.

**Comment by GeorgeARM** [comment]
Resolved in 18.02. Reopen if needed
